TransTest.hs

```diff
@@ -0,0 +1,131 @@
+import System.Exit (exitFailure)
+
+import Data.List
+
+import Tuura.Concept.STG
+import Tuura.Concept.STG.Translation
+
+main :: IO ()
+main = do
+    testDotGOutput
+    testHandshakeConcept
+    testOrGateConcept
+    testTwoAndGates
+    testTwoDifferentCElements
+
+testDotGOutput :: IO ()
+testDotGOutput = do
+    putStrLn "=== testDotGOutput"
+    assertEq threeSignalsConcept expected
+  where
+    expected = ".model out\n"          ++
```

Comment:
Not that this is a bit difference, but I think `unlines $ [ ".model out", ...  ]` is typically easier to work with.